File: kitnet/src/main/java/kitnet/com/api/dto/apartamento/ApartamentoResponseDTO.java
package kitnet.com.api.dto.apartamento;

import kitnet.com.api.dto.predio.PredioId;

public record ApartamentoResponseDTO(
    Long id,
    String numeroApartamento,
    String descricao,
    String medidor,
    String statusApartamento,
    PredioId predio
) {}

File: sis-kitnet/src/app/core/model/dto/apartamento/apartamentoResponseDTO.ts
export interface PredioIdDTO {
  id: number;
}

export interface ApartamentoResponseDTO {
  id: number;
  descricao: string;
  numeroApartamento: string;
  medidor?: string | null;
  statusApartamento: string;
  predio: {
    id: number;
  };
}


